src/commands/aksKaito/aksKaitoCreateCRD.ts

```diff
@@ -12,8 +12,22 @@ import { getExtension, longRunning } from "../utils/host";
 
 export default async function aksKaitoCreateCRD(_context: IActionContext, target: unknown): Promise<void> {
     const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+    if (!cloudExplorer.available) {
```

Comment:
cloud explorer check is repeated, can you remove one?

Reply:
The second one was deleted 👍, moved things around .